spec/unit/lib/cloud_controller/paging/sequel_paginator_spec.rb

```diff
@@ -117,6 +119,38 @@ module VCAP::CloudController
         expect(paginated_result.total).to be 1
       end
 
+      it 'does not order by GUID when the primary order is by ID' do
+        options = { page: page, per_page: 2, order_by: 'id', order_direction: 'asc' }
+
+        pagination_options = PaginationOptions.new(options)
+        expect {
+          paginator.get_page(dataset, pagination_options)
+        }.to have_queried_db_times(/.*ORDER BY `apps`.`id` ASC, `apps`.`guid` ASC.*/i, 0)
```

Comment:
I would add a positive test
```
... have_queried_db_times(/ORDER BY `apps`.`id` ASC/i, 1)
```
to ensure that the SQL string is correct for all tested databases. I remember that formulating a regex that matches all databases was difficult...